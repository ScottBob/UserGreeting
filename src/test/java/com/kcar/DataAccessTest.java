package com.kcar;

import org.approvaltests.Approvals;
import org.junit.Test;

import java.util.Map;

public class DataAccessTest {
    @Test
    public void getDataTest() {
        DataAccess da = new DataAccess();
        Map<String, String> data = da.getData();
        Approvals.verify(data);
    }
}
