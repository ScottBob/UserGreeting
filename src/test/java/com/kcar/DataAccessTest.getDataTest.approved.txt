CPU : 256 
MEM : 512m 
Zane : SuperSmart 
